<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginNetSetWindow</class>
 <widget class="QWidget" name="LoginNetSetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>334</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>428</width>
    <height>334</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>428</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginNetSetWindow</string>
  </property>
  <widget class="QPushButton" name="pButtonOk">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>305</y>
     <width>84</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonCancel">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>305</y>
     <width>84</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNet">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="imageTestBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chatTestBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileTestBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>聊天服务:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chatAddr">
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图片服务:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imageAddr">
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>文件服务:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileAddr">
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
